src/hooks/useTabs/TabStyles.module.css

```diff
@@ -99,9 +99,9 @@
   overflow: hidden;
 }
 
-.clippedOverflowX {
+.clippedOverflow {
   overflow-x: hidden;
-  overflow-y: visible;
+  overflow-y: hidden;
```

Comment:
@piazzatron just in case you can think of anything this breaks. let's soak this change on stage for a while & go through weekend QA.
@sliptype might be worth testing mobile layout using the devtools thing

Reply:
Yeah everything looks good on mobile too, I'll go ahead and merge and we can keep our eyes peeled in staging